js/main.js

```diff
@@ -2,16 +2,19 @@ import {createOffers} from './data.js';
 
 import {createPopup} from './popup.js';
 
-import {disableСreateForm, disableFilterForm, enableСreateForm, enableFilterForm} from './state.js';
+import {disableAdForm, disableFilterForm, enableAdForm, enableFilterForm} from './state.js';
+
+import './ad-form.js';
```

Comment:
Лучше всегда контролировать поведение внутри модулей. Вдруг тебе вообще не понадобится валидация, если ты вообще не разблокируешь форму по какой-то причине? Тогда этот код выполнится зря.

```suggestion
import {initValidation} from './ad-form.js';
```